tests/resources/dsl/without_policies.yaml

```diff
@@ -0,0 +1,26 @@
+# this blueprint is like with_policies1, but without the policies
+# - so that we have two simply blueprints that only differ by the policies
```

Comment:
"simple"? :)

ill also use this comment to also mention that the CI tests are failing